target/linux/bcm27xx/bcm2712/config-6.12

```diff
@@ -583,6 +583,14 @@ CONFIG_SPARSEMEM_EXTREME=y
 CONFIG_SPARSEMEM_VMEMMAP=y
 CONFIG_SPARSEMEM_VMEMMAP_ENABLE=y
 CONFIG_SPARSE_IRQ=y
+CONFIG_SPI=y
+# CONFIG_SPI_BCM2835AUX is not set
+CONFIG_SPI_DESIGNWARE=y
+# CONFIG_SPI_DW_DMA is not set
+CONFIG_SPI_DW_MMIO=y
+# CONFIG_SPI_DW_PCI is not set
+CONFIG_SPI_DYNAMIC=y
+CONFIG_SPI_MASTER=y
```

Comment:
I would prefer if you build this as a loadable kernel module in `package/kernel/linux/modules/spi.mk`.

Reply:
Thanks for reviewing! I will do it.